spec/face.html

```diff
@@ -1098,6 +1098,153 @@
+        
+                ft.onready = function(e) {
+                  console.log('Face module ready to start');
+                  // The stream got ready, we can start face module now.
+                  ft.start().then(
+                      function() {
+                        console.log('Face module start succeeds');},
+                      function(e) {
+                        console.log('Face module start failed: ' + e.message);}); 
+                };
+        
+                ft.onprocessedsample = function(e) {
+                  console.log('Got face module processedsample event.');
+                  ft.getProcessedSample(false, false).then(function(processedSample) {
+                    console.log('Got face module processedsample data.');
+                    // Use processedSample.faces data to work for you.
+                    console.log('Detected faces number: ' + processedSample.faces.length);
+                    // You can get all avaiable detection/landmarks/recognition data
+                    // of every face from processedSample.faces.
+                  }, function(e) {
+                    console.log('Failed to get processed sample: ' + e.message);});
+                };
+        
+                ft.onerror = function(e) {
+                  console.log('Got face module error event: ' + e.message);
+                };
+        
+                ft.onended = function(e) {
+                  console.log('Face module ends without stop');
+                };
+        
+                ft.ready = false;
+                ftStarted = false;
+                onGetConfButton();
+              }, errorCallback);
+        };
+
+        function stopPreviewStream() {
+          if (previewStream) {
+            previewStream.getTracks().forEach(function(track) {
+              track.stop();
+            });
+            if (ft) {
+              // Remove listeners as we don't care about the events.
+              ft.onerror = null;
+              ft.onprocessedsample = null;
+              ft = null;
+            }
+          }
+          previewStream = null;
+        }
+
+        // Stop face module and stream.
+        stopButton.onclick = function(e) {
+          if (!ft) return;
+          ft.stop().then(
+              function() {
+                console.log('Face module stop succeeds');
+                stopPreviewStream();},
+              function(e) {
+                console.log('Face module stop failed');
+                stopPreviewStream();});
+        };
+      </pre>
+      <h3>
+        Face module configuration.
+      </h3>
+      <pre class='example highlight'>
+        var setConfButton = document.getElementById('setConfButton');
+        var getConfButton = document.getElementById('getConfButton');
+        var getDefaultConfButton = document.getElementById('getDefaultConfButton');
+
+        // Set configuration. Simple configuration example as bellow.
+        var confData = {
```

Comment:
How about adding comment like "Please refer to FaceConfigurationData interface for confData details."